Automattic/wp-calypso
# Stroke "Write your first post" in the writing flow

Related to 2216-gh-Automattic/dotcom-forge

Asked for desing input here: p1682461024801329-slack-CKZHG0QCR

## Proposed Changes

* Stroke "Write your first post" in the writing flow

<img width="1031" alt="image" src="https://user-images.githubusercontent.com/1044309/234415512-9308439c-bf34-40a3-992f-7418286ee19f.png">


## Testing Instructions



* Using an unlaunched site, access the start-writing flow: http://calypso.localhost:3000/setup/start-writing/launchpad?siteSlug={yourUnlaunchedSiteDomain}
* Ensure the user can't get back to the post by clicking at `Write your first post`:
* Switch to the normal 'write' flow: http://calypso.localhost:3000/setup/write/launchpad?siteSlug={yourUnlaunchedSiteDomain}
* Ensure user can click normally
* Ensure tests are passing with this command: `yarn run test-client:watch`

## Pre-merge Checklist
